src/ObjectUtils/include/ObjectUtils/SymbolsFile.h

```diff
@@ -45,6 +46,14 @@ class SymbolsFile {
 ErrorMessageOr<std::unique_ptr<SymbolsFile>> CreateSymbolsFile(
     const std::filesystem::path& symbol_file_path, const ObjectFileInfo& object_file_info);
 
+// The file extensions for symbols files are .debug for elf files and .pdb for coff files. Only
+// files with following formats are considered `module.sym_ext`, `module.mod_ext.sym_ext` and
+// `module.mod_ext`. (`mod_ext` is the module file extension, usually .elf, .so, .exe or .dll;
+// `sym_ext` is either .debug or .pdb)
+[[nodiscard]] std::vector<std::filesystem::path> GetStandardSymbolFilenamesForModule(
```

Comment:
On my side, I feel like this should be in a different "utils" file as this file is specifically for the `SymbolsFile` class.

Reply:
As discussed offline, I moved it into `Symbols/SymbolUtils`.